application/templates/admin/merge_data_sources.html

```diff
@@ -16,7 +16,7 @@
       <h1 class="govuk-heading-xl">Merge data sources</h1>
 
       <div class="govuk-body">All of the following data sources will be merged together.</div>
-      <form action="/data-sources/merge" method="POST">
+      <form action="/admin/data-sources/merge" method="POST">
```

Comment:
Can we switch this to using `url_for`? I don't think we want to be putting URLs directly in here.